common/src/ray_tracing.rs

```diff
@@ -0,0 +1,105 @@
+use crate::math;
+
+/// Performs ray tracing against the z=0 plane in node coordinates. This plane is one-sided, so the ray must be going in the -z
+/// direction to detect an intersection.
```

Comment:
It would be the -z direction based on the Hyperboloid model. This would also be the direction represented by the `(0, 0, -1, 0)` vector, which is a valid direction vector, as it's tangent to the hyperboloid whenever we're on the z=0 plane.